collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/DeleteBlankFormFragment.kt

```diff
@@ -42,6 +44,8 @@ class DeleteBlankFormFragment(
             }
         )
     }
+    private var appBarLayout: AppBarLayout? = null
```

Comment:
These fields should be removed, and the views should be accessed via `OfflineMapLayersPickerBinding.bind(requireView())` (or just `view` in `onViewCreated`) so we don't leak anything after `onDestroy`.

Reply:
I can't do that as `appBarLayout` is not a part of the layout used in this fragment (the fragment is included using `viewpager`) and the same with the ist that should be watched. I also can't move this to `onViewCreated` because: https://github.com/getodk/collect/pull/6390#discussion_r1745281852

Reply:
The interaction between `ViewPager2` and `AppBarLayout` here are really annoying! It feels like this is probably something that's missing from the `MenuProvider` system right now. So to sum it up: we want to set the `RecyclerView` child of a fragment (which is in a child fragment) as the scroll target view for an `AppBarLayout` when the fragment is shown in the view page right? I think we can do that generally in the `onResume` of `MultiSelectListFragment`:


```kotlin
override fun onResume() {
    super.onResume()
    
    val list = MultiSelectListBinding.bind(requireView()).list
    val appBarLayout =
        requireActivity().findViewById<AppBarLayout>(org.odk.collect.androidshared.R.id.appBarLayout)
    appBarLayout?.setLiftOnScrollTargetView(list)
}
```

We could probably make this a bit more configurable by passing in an optional app bar layout id to `MultiSelectListFragment`'s constructor, but I think this sort of approach will work well with the "configuration over convention" choice to always have `appBarLayout` use the same id.

I think there's also ways we could go further to generalize it using lifecycle observers similarly to how `MenuHost` is implemented on `ComponentActivity`, but I think we can leave that to Google.

Reply:
Done.